src/lib/awsManager.js

```diff
@@ -216,9 +302,11 @@ module.exports = {
   constructParamPath,
   getParameter,
   getParametersByService,
+  getSharedConfigByService,
+  putSharedConfigRecordsByService,
   setParameter,
   setParametersByService,
   getEnvironments,
   getServicesForEnvironment,
   getAllOrgParams
-}
+}
```

Comment:
You would need to update `README` file too - with the 2 new commands

Reply:
This bit is still pending

Reply:
updated thanks.